server/src/main/kotlin/com/mobilejazz/colloc/CollocApplication.kt

```diff
@@ -12,12 +15,16 @@ import org.springframework.stereotype.Controller
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.RequestParam
 
+
 @SpringBootApplication
 class CollocApplication
 
 @Controller
 class CollocController {
 
+  @Autowired
+  lateinit var beanFactory: BeanFactory
+
```

Comment:
Not needed, you can remove it. 

Read message in the `LocalizationAndroidEncoder`